logical_switch_port.go

```diff
@@ -193,16 +193,30 @@ func (odbi *ovndb) lspGetDHCPv6OptionsImp(lsp string) (*DHCPOptions, error) {
 }
 
 func (odbi *ovndb) lspSetOptionsImp(lsp string, options map[string]string) (*OvnCommand, error) {
-	mutatemap, _ := libovsdb.NewOvsMap(options)
-	mutation := libovsdb.NewMutation("options", opInsert, mutatemap)
+	if options == nil {
+		return nil, ErrorOption
+	}
+
+	if len(lsp) == 0 {
```

Comment:
Why include lsp validations when adding nb/sb global db support? Can you move this validation in separate commit?

Reply:
Will do. Ack.

Reply:
Btw, these are not just validations. The implementation of the `SetOptions` and `SetExternalIds` was not correct. The mutate operation would fail if the map argument contained options/external ids already present in the db.